pages/configuring_webif.rst

```diff
@@ -182,6 +182,31 @@ If you want to use nginx to proxy access to a Graylog server, you have several o
     }
   }
 
+Apache:
+------- 
+
+For the following samples we are assuming that your Graylog instance is running on ``graylog.internal.example.org`` using the default ports of 12900 for the REST API and 9000 for the web interface. SSL is disabled for both. You want to expose the Graylog web interface as ``https://graylog.example.org``. 
+
+**URI Configs in Graylog server conf**::
+
+   rest_listen_uri = http://127.0.0.1:12900/
+   web_listen_uri = http://127.0.0.1:9000/
+   web_endpoint_uri = https://graylog.example.org/api/
```

Comment:
I would prefer doing this with the `X-Graylog-Server-URL`-header, because it makes the setup more flexible (you can point multiple proxies to it, exposing the web interface under different URLs). What do you think?